tests/test_web_log.py

```diff
@@ -214,3 +216,33 @@ class Logger(AbstractAsyncAccessLogger):
     resp = await cli.get('/path/to', headers={'Accept': 'text/html'})
     assert resp.status == 200
     assert msg == '/path/to: 200'
+
+
+@pytest.mark.skipif(not PY_37,
+                    reason="contextvars support is required")
+async def test_contextvars_logger(aiohttp_server, aiohttp_client):
+    from contextvars import ContextVar
```

Comment:
plz put this at the top of this test module using `try/import/except ImportError` style

Reply:
will do